client/src/components/CustomerDetails.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { gql, useMutation } from '@apollo/client';
+
+import { useHistory } from "react-router-dom";
+
+const CUSTOMER_DETAILS = gql`
+    mutation EditCustomer($friendlyName:String! $phoneNumber:String!) {
```

Comment:
calling it `EditCustomer` when it actually creates a customer is confusing.

Reply:
You are right. I will modify the var's name to `CUSTOMER_CREATE` , and the mutation's name to `CreateCustomer`,  are they ok for you?

Reply:
Sounds good, another thing is that this variable represents a GQL query, so your variable name should reflect the fact that it's a GQL statement.

Reply:
You can also look at examples from the GQL react tutorials to get some ideas about naming variables. Good variables significantly improve readability and it takes practice to find good names.

Reply:
I've already changed all the names which refer to "customer edit" and "customer detail" to "customer create" related names on my local file, since this component is now just used for creating a new customer. I think you will see the names updated on the further commits maybe from the PR about "deleting customer".

Reply:
This is the branch that's about to go into main, so this problem needs to be addressed in this PR, not future ones.

Reply:
I just made a new commit on this branch for all the naming problems. You can check it now.